### PR TITLE
Run dynamic simulation

### PR DESCRIPTION
**Pull request by thangqp**

### CODE REVIEW

**Comment by thangqp** [comment]
image

**Comment by thangqp** [comment]
image

**Comment by thangqp** [comment]
image

File: src/components/results/dynamicsimulation/plot/plotly-series-chart.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    rightSeries,
+    sync,
+}) => {
+    const [layout, setLayout] = useState(
+        JSON.parse(JSON.stringify(defaultLayout)) // deep clone can be done by lodash
+    );
+    //const [revision, setRevision] = useState(0);
+
+    const makeGetMarker = useMemo(
+        () => (opts) => {
```
Comment (jonenst) [inline src/components/results/dynamicsimulation/plot/plotly-series-chart.js:27]: useCallback

Comment (thangqp) [reply src/components/results/dynamicsimulation/plot/plotly-series-chart.js]: Done! Thanks

**Review by jonenst — APPROVED** [review]

**Pull request merged**